lib/svg_types.mli

```diff
@@ -279,6 +279,8 @@ type strings = string list
 type color = string
 type icccolor = string
 
+type opacity = float
```

Comment:
Can you add a comment that it's intended to be between O and 1 ? Same on the various `a_*_opacity` functions.